var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -480,6 +485,18 @@ def cmake_args(self):
                 '-DTrilinos_ENABLE_SEACASExodus:BOOL=OFF'
             ])
 
+        if '+chaco' in spec:
+            options.extend([
+                '-DTrilinos_ENABLE_SEACAS:BOOL=ON'
+                '-DTrilinos_ENABLE_SEACASChaco:BOOL=ON'
+            ])
+        else:
+            # don't disable SEACAS, could be needed elsewhere
+            options.extend([
+                '-DTrilinos_ENABLE_SEACASChaco:BOOL=OFF',
+                '-DTrilinos_ENABLE_SEACASNemslice=OFF'
```

Comment:
Just to verify: is this line necessary? Does `SEACASNemslice` depend on `SEACASChaco`?

Reply:
yes, Trilinos config complained otherwise.